### PR TITLE
added toInt function

### PR DESCRIPTION
**Pull request by fro0m**

### CODE REVIEW

File: HttpStatusCodes_C++11.h [hunk 1]
```diff
@@ -111,6 +111,13 @@ enum class Code
 	NetworkAuthenticationRequired = 511  //!< Indicates that the client needs to authenticate to gain network access.
 };
 
+/*! Explicitly converts code to int
```
Comment (j-ulrich) [inline HttpStatusCodes_C++11.h:114]: The parameter and return value should be formally documented:
```suggestion
/*! Converts a Code to its corresponding integer value.
 * \param code The code to be converted.
 * \return The numeric value of \p code.
```

**Review by j-ulrich — COMMENTED** [review]
Thank you for your contribution! I appreciate that!

Let me know if you can and want to take care of the review comments. Else I will take care of them.

File: HttpStatusCodes_C++11.h [hunk 2]
```diff
@@ -111,6 +111,13 @@ enum class Code
...
+ */
+int toInt(Code code)
```
Comment (j-ulrich) [inline HttpStatusCodes_C++11.h:116]: The `toInt()` method should also get a unit test.

**Review by j-ulrich — CHANGES_REQUESTED** [review]
The added feature should be documented in the `CHANGELOG.md` like this:
```
---


## Unreleased ##

### Added ###
- [#5] toInt() method to convert HttpStatus::Code to integer in C++11 variant.


---
```

And in the `README.md` (replacing the "QNetworkReply::NetworkError Mapping" section):

    ### Conversion Functions ###
    
    #### C++11 Variant ####
    
    ```c++
    int HttpStatus::toInt(Code code);
    ```
    Returns the numerical value of a given `HttpStatus::Code`. This method is provided for convenience to avoid writing a `static_cast<int>()`.
    
    #### Qt Variant ####
    > **Note:** Available only if the Qt::Network module is available.
    ```c++
    int HttpStatus::networkErrorToStatusCode(QNetworkReply::NetworkError error);
    ```
    Returns the HTTP status code corresponding to the given _error_ if there is one.
    Otherwise, `-1` is returned.
    
    ```c++
    QNetworkReply::NetworkError HttpStatus::statusCodeToNetworkError(int code);
    ```
    Returns the `QNetworkReply::NetworkError` corresponding to the given _code_ if there is one.
    For codes where there is no exact match, the best matchnig "catch all" code (`QNetworkReply::NoError`,
    `QNetworkReply::UnknownContentError`, `QNetworkReply::UnknownServerError` or `QNetworkReply::ProtocolFailure`)
    is returned.
    ```

**Comment by fro0m** [comment]
Please take care of unit-tests and better to make the function explicitly inline.

**Comment by j-ulrich** [comment]
Closing this PR in favor of #6 which also takes care of the review comments.

Thanks @fro0m for the contribution!

**Pull request closed without merging**